client/login/controller.js

```diff
@@ -12,15 +12,16 @@ import HandleEmailedLinkForm from './magic-login/handle-emailed-link-form';
 
 export default {
 	login( context, next ) {
-		const { lang, path, params: { flow, twoFactorAuthType } } = context;
+		const { lang, path, params: { flow, twoFactorAuthType }, query } = context;
 
 		context.primary = (
 			<WPLogin
 				locale={ lang }
 				path={ path }
 				twoFactorAuthType={ twoFactorAuthType }
 				socialConnect={ flow === 'social-connect' }
-				privateSite={ flow === 'private-site' } />
+				privateSite={ flow === 'private-site' }
+				clientId={ query.clientId } />
```

Comment:
For a super lazy reference:

```
$ find . -name 'controller.js' | xargs ack 'query\.[a-z]+\b' | wc -l
      31
$ find . -name 'controller.js' | xargs ack 'query\.[a-z]+_[a-z_]*\b' | wc -l
       3
$ find . -name 'controller.js' | xargs ack 'query\.[a-z]+-[a-z-]*\b' | wc -l
       0
$ find . -name 'controller.js' | xargs ack 'query\.[a-z]+[A-Z][a-zA-Z]*\b' | wc -l
       2
```

So, underscore or camelCase seem to be the options, hyphen is never used.